amundsen_application/static/js/components/DashboardPage/index.tsx

```diff
@@ -172,23 +173,29 @@ export class DashboardPage extends React.Component<DashboardPageProps, Dashboard
         </header>
         <article className="column-layout-1">
           <section className="left-panel">
-            <div className="section-title title-3">Description</div>
-            {
-              hasDescription &&
-              <div>
-                { dashboard.description }
-              </div>
-            }
-            {
-              !hasDescription &&
-              <a
-               className="edit-link body-2"
-               target="_blank"
-               href={ dashboard.url }
-              >
-               {`${ADD_DESC_TEXT} ${getSourceDisplayName(dashboard.product, ResourceType.dashboard)}`}
-              </a>
-            }
+            <EditableSection
+              title="Description"
+              readOnly={ true }
+              editUrl={ dashboard.url }
+              editText={`${EDIT_DESC_TEXT} ${getSourceDisplayName(dashboard.product, ResourceType.dashboard)}`}
+            >
+              {
+                hasDescription &&
+                <div>
+                  { dashboard.description }
+                </div>
+              }
+              {
+                !hasDescription &&
+                <a
+                 className="edit-link body-2"
+                 target="_blank"
```

Comment:
I found it is a good practice to add the rel=”noopener noreferrer” attribute to all links that have `target="_blank"`. 

Maybe it is not strictly applicable to this case in particular, but either way is good to just add it always. Check [more info here](https://web.dev/external-anchors-use-rel-noopener/).